daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -87,7 +87,7 @@ class RazerNostromo(_RazerDeviceBrightnessSuspend):
                # ?
                'keypad_get_mode_modifier', 'keypad_set_mode_modifier']
 
-    DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/564/564_tartarus_classic.png"
+    DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/564/Blade14-Piper-FHD.png"
```

Comment:
This change seems to have been snuck in after running the linter. The link is broken and it seems to be unintended? This should probably be reverted.

Reply:
If you make a pull request on my fork I'll merge it quickly, otherwise I'll make the change later

Reply:
@BeanieBen9990 Done 👍